### ISSUE TITLE
Include path in file reference

### ISSUE THREAD

**Issue by varac** [terraform-linters/tflint#991]
When referencing files with linter warnings please include the path which would make it *much* easier to locate these files.
I'm using [pre-commit](https://pre-commit.com/) together with [antonbabenko/pre-commit-terraform](https://github.com/antonbabenko/pre-commit-terraform) and this hooks changes to different modules directories and you don't have any idea which file the output is complaining, i.e.:

```
Warning: Module should include an empty outputs.tf file (terraform_standard_module_structure)

  on outputs.tf line 1:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint/blob/v0.21.0/docs/rules/terraform_standard_module_structure.md

1 issue(s) found:

Warning: Module should include an empty outputs.tf file (terraform_standard_module_structure)

  on outputs.tf line 1:
   (source code not available)

Reference: https://github.com/terraform-linters/tflint/blob/v0.21.0/docs/rules/terraform_standard_module_structure.md
```

**Comment by bendrucker** [comment]
How are you invoking TFLint? If you're running it like this that's a bug:

```sh
tflint path/to/my/dir
```

If you're invoking it like this, it's expected:

```sh
cd path/to/my/dir && tflint .
```

**Comment by bendrucker** [comment]
As you noted in the other issue, if you do the latter, you'd need some sort of header for the working directory. 

If you pass each module path to TFLint, you'd need to do the same with `terraform init`.

**Comment by varac** [comment]
the pre-commit hook does the latter. that's why it's not a bug, more a useful nice-to-have, but I'd be fine fixing this in the pre-commit hook.

**Comment by bendrucker** [comment]
Yeah, probably not something we're likely to change in a way that makes this problem go away. The choices are to print the file paths relative to the process working directory or print absolute paths. Relative paths are best for readability, and adding flags to convert those to absolute is a pain. AFAIK this is also how Terraform would behave (e.g. `terraform validate`) and so the same choices of printing a header in a monorepo or passing the module paths as an arg apply.

**Comment by 1oglop1** [comment]
Thank you this is amazing! I hope it gets merged in the hook soon!

**Comment by bendrucker** [comment]
Cool, doesn't sound like there are any TFLint changes to make here.
